tests/spec/core/utils-spec.js

```diff
@@ -612,5 +612,71 @@ describe("Core - Utils", () => {
         expect(utils.getElementIndentation(anchor)).toBe("          ");
       });
     });
+
+    describe("toMDCode()", () => {
+      it("wraps a string in backticks", () => {
+        expect(utils.toMDCode("")).toBe("");
+        expect(utils.toMDCode("test")).toBe("`test`");
+      });
+    });
+
+    describe("mdJoinOr()", () => {
+      it("uses uses disjunction", () => {
+        expect(utils.mdJoinOr([])).toBe("");
+        expect(utils.mdJoinOr(["a", "b", "c"])).toBe("`a`, `b`, or `c`");
+      });
+      it("quotes and uses disjunction", () => {
+        expect(utils.mdJoinOr([], { quotes: true })).toBe("");
+        expect(utils.mdJoinOr(["a", "b", "c"], { quotes: true })).toBe(
+          '`"a"`, `"b"`, or `"c"`'
+        );
+      });
+    });
+
+    describe("mdJoinAnd()", () => {
+      it("uses conjunction", () => {
+        expect(utils.mdJoinAnd([])).toBe("");
+        expect(utils.mdJoinAnd(["a", "b", "c"])).toBe("`a`, `b`, and `c`");
+      });
+      it("quotes and uses conjunction", () => {
+        expect(utils.mdJoinAnd([], { quotes: true })).toBe("");
+        expect(utils.mdJoinAnd(["a", "b", "c"], { quotes: true })).toBe(
+          '`"a"`, `"b"`, and `"c"`'
+        );
+      });
+    });
+    describe("docLink", () => {
+      const docLink = utils.docLink;
+      it("it allows unlinked strings", () => {
+        const result = docLink`Link to ${"nothing"}.`;
+        expect(result).toBe("Link to nothing.");
```

Comment:
```suggestion
        const result = docLink`Link to ${"nothing"} ${1+2}.`;
        expect(result).toBe("Link to nothing 3.");
```

Reply:
We probably don't (want to) support this?

Reply:
This might throw, so I'll avoid this for now.